### DISCUSSION TITLE
Opendtu on battery Zendure

### DISCUSSION THREAD

**Discussion by Matze1083** [hoylabs/OpenDTU-OnBattery#1308] [Q&A] [answered]
Wäre es möglich Zendure solarflow in die Opendtu on battery als Auswahl einzusetzen so wie es bei Victon auch schon gibt. Lg

**Comment by schlimmchen** [comment]
Ist #1221 was du willst?

**Reply by Matze1083** [reply]
Ja so ähnlich. Mit cloud und Anbindung an die Opendtu wäre geil.

**Reply by vaterlangen** [reply]
Da ich mein System nur an meinem lokalen Broker betreibe, gibt es für mich den Use-Case Vendor Cloud nicht. Je nachdem, was da genau aus der Cloud raus kommt, ließe sich das aber auch mit relativ wenig Aufwand dahingehend erweitern. Wie ich es aus div. Erfahrungsberichten herauslesen, ist die Cloudanbindung allerdings Recht langsam - kann ich aber nicht selbst bewerten.

Falls du einen lokalen Broker sowie DNS Resolver laufen hast und es mal mit #1221 probieren möchtest , meld dich.

**Reply by ghost** [reply]
Hallo @vaterlangen, ich habe aktuell den zendure hub über einen iobroker-adapter angebunden und würde gern mal #1221 ausprobieren. Ist das auch mit der Vers. 2024.06.03 möglich ?

**Reply by vaterlangen** [reply]
@Negruej der Branch baut auf dem aktuellen Development Stand auf - wenn dein Speicher zu klein ist, geht das leider nur ohne OTA Update, wie in  #1025 beschrieben. Ist dein HUB schon Cloud-Frei, oder geht deine Anbindung aktuell über die Zendure Server (das wäre dann aktuell nicht unterstützt)

**Reply by ghost** [reply]
@vaterlangen Danke für die schnelle Info, dann update ich erst mal meine HW/SW und melde mich nochmals wenn das wieder läuft. LG

**Reply by Matze1083** [reply]
@ vaterlangen. Gerne würde ich es mal ausprobieren. Hab einen Broker auf homeassistant laufen.  Aber das update von die konnte ich nicht aufs s3 bekommen, es kommt immer eine Fehlermeldung. Lg Matze

**Reply by vaterlangen** [reply]
Welche Fehlermeldung? Und welchen FW Stand verwendest du aktuell?
 Bei mir läuft es seit Wochen auf einem OpenDTUFusionv2 (inkl. Updates über die WebUI)

**Reply by Matze1083** [reply]
![Screenshot_20241008_133229_Chrome](https://github.com/user-attachments/assets/3ffc18c1-f54c-4017-8990-b5214a172cbb)

OTA Fehler  Scheitern kommt immer.

**Reply by schlimmchen** [reply]
Klingt danach als hättest du dein Partitionslayout noch nicht aktualisiert. [Doku](https://opendtu-onbattery.net/firmware/howto/upgrade_8mb/) schon gelesen?

**Comment by Matze1083** [comment]
@vaterlangen  besteht die Möglichkeit das im Display der DTU die Batterie % Anzeige zu sehen ist ? Des wäre echt cool 😎 lg

**Reply by vaterlangen** [reply, marked as answer]
Hab selber kein Display - hätte jetzt erwartet, das es dort angezeigt wird...

Läuft denn sonst soweit alles bei dir?

**Reply by Matze1083** [reply]
Ok. Hab mein S3 neu geflasht jetzt läuft es mit deiner Software. Ne die Batterie wird nicht in % im Display angezeigt. Lg

**Reply by vaterlangen** [reply]
Der Ladezustand geht auch über das Batterieinterface - dann wäre das vmtl ein genereller Feature Request an das Projekt.
